src/test/java/io/confluent/examples/streams/interactivequeries/kafkamusic/KafkaMusicExampleTest.java

```diff
@@ -225,12 +226,44 @@ public void shouldCreateChartsAndAccessThemViaInteractiveQueries() throws Except
 
     // Wait until the all-songs state store has some data in it
     TestUtils.waitForCondition(() -> {
-      final ReadOnlyKeyValueStore<Long, Song>
-          songsStore;
+      final ReadOnlyKeyValueStore<String, KafkaMusicExample.TopFiveSongs> topFiveStore;
+      final ReadOnlyKeyValueStore<String, KafkaMusicExample.TopFiveSongs> topFiveByGenreStore;
```

Comment:
It looks like these aren't used outside of the try/catch, so I think you can declare them at initialization.